stable/ignite/values.yaml

```diff
@@ -0,0 +1,50 @@
+# Default values for ignite.
+replicaCount: 2
+
+image:
+  repository: apacheignite/ignite
+  tag: 2.7.0
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
+fullnameOverride: ""
+
+rbac: yes
+
+env:
+  OPTION_LIBS: "ignite-kubernetes,ignite-rest-http"
+  IGNITE_QUIET: "false"
+  JVM_OPTS: "-Djava.net.preferIPv4Stack=true"
+peerClassLoadingEnabled: false
+dataStorage:
```

Comment:
In most charts they use `persistence` as the key here:
https://github.com/helm/charts/blob/master/stable/wordpress/values.yaml#L228
https://github.com/helm/charts/blob/master/stable/redis/values.yaml#L190

Reply:
This is fixed